### PR TITLE
Add portuguese translation of 0.21.2

### PR DESCRIPTION
**Pull request by mateusnds**
Hello everyone, I intend to help translate bitcoincore to Portuguese, I'm starting with this file to see if everything goes well.

### CODE REVIEW

**Comment by katesalazar** [comment]
will review

**Comment by katesalazar** [comment]
missing the language name

---
Screen Shot 2022-02-22 at 20 55 00

---

File: _posts/pt_BR/posts/2021-09-13-release-22.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement.  English posts only
+announcement: 0
+
+excerpt: >
+  Bitcoin Core 0.21.2 Está disponível.
+---
+A versão 0.21.2 do Bitcoin Core está disponível para
```
Comment (katesalazar) [inline _posts/pt_BR/posts/2021-09-13-release-22.0.md:19]: nit: extra whitespace here

File: _posts/pt_BR/posts/2021-09-13-release-22.0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Bitcoin Core 0.21.2 lançado
+name: blog-release-0.21.2
+id: pt-BR-blog-release-0.21.2
```
Comment (katesalazar) [inline _posts/pt_BR/posts/2021-09-13-release-22.0.md:4]: not matching the document name (...-22.0.md)

**Review by katesalazar — CHANGES_REQUESTED** [review]
- inconsistent name and content?
- lang name missing in the selector
- not passing ci

**Comment by fanquake** [comment]
Please consolidate your changes into a single PR. i.e #905. Rather than opening many different ones.

**Pull request closed without merging**